crates/holochain/src/conductor/chc/chc_local.rs

```diff
@@ -300,55 +353,47 @@ mod tests {
             .await
             .unwrap();
 
+        // Sync is not be possible since the installation was rolled back and the cell was removed
+        assert!(matches!(
+            conductors[3]
+                .raw_handle()
+                .chc_sync(cell_id.clone(), None)
+                .await,
+            Err(ConductorApiError::ConductorError(ConductorError::CellMissing(id))) if id == *cell_id
+        ));
+
         let dump1 = conductors[1]
             .dump_full_cell_state(&cell_id, None)
             .await
             .unwrap();
 
         assert_eq!(dump1.source_chain_dump.records.len(), 3);
 
-        let _apps1 = conductors[1]
-            .setup_app_for_agent("app", agent.clone(), [&dna_file])
-            .await
-            .unwrap();
-        let _apps2 = conductors[2]
-            .setup_app_for_agent("app", agent.clone(), [&dna_file])
-            .await
-            .unwrap();
-
         let c1: SweetCell = conductors[1].get_sweet_cell(cell_id.clone()).unwrap();
         let c2: SweetCell = conductors[2].get_sweet_cell(cell_id.clone()).unwrap();
 
         let _: ActionHash = conductors[0]
-            .call(
-                &c0.zome(SweetInlineZomes::COORDINATOR),
-                "create_string",
-                AppString::new("zero"),
-            )
+            .call(&c0.zome(TestWasm::Create), "create_entry", ())
             .await;
 
+        conductors[1].enable_app("app".into()).await.unwrap();
+        conductors[2].enable_app("app".into()).await.unwrap();
+
         // This should fail and require triggering a CHC sync
         let hash1: Result<ActionHash, _> = conductors[1]
-            .call_fallible(
-                &c1.zome(SweetInlineZomes::COORDINATOR),
-                "create_string",
-                AppString::new("one"),
-            )
+            .call_fallible(&c1.zome(TestWasm::Create), "create_entry", ())
             .await;
 
         dbg!(&hash1);
 
         regex::Regex::new(
-            r#".*ChcHeadMoved\("SourceChain::flush", InvalidChain\(4, ActionHash\([a-zA-Z0-9-_]+\).*"#
+            r#".*ChcHeadMoved\("SourceChain::flush", InvalidChain\((\d+), ActionHash\([a-zA-Z0-9-_]+\).*"#
         ).unwrap().captures(&format!("{:?}", hash1)).unwrap();
+        // TODO: check sequence and hash
 
         // This should trigger a CHC sync
```

Comment:
It's not part of this PR, but this suggests a different response than the same operation on conductor 1. The two responses are asserted to be equal then, so it must both be errors. A bit confusing the comment, that's all =)